app/src/ui/organism/hero-section/HeroSection.js

```diff
@@ -0,0 +1,84 @@
+import Image from "../../atoms/imagine/Image";
+import HeroImg1 from "../../../assets/img/hero-img-1.png";
+import HeroImg2 from "../../../assets/img/hero-img-2.png";
+import HeroImg3 from "../../../assets/img/hero-img-3.png";
+import Slider from "react-slick";
+
+const HeroSection = () => {
+  const settings = {
+    dots: true,
+    fade: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    waitForAnimate: false,
+  };
+  return (
+    <div className="hero-section">
```

Comment:
.hero-section is just hero-bg.png background - this is not mandatory to change but I think that better is to create .bg-hero class - this tell us what is responsibility of this class

Reply:
done